cfn/templates/ecs-task.template.yml

```diff
@@ -1,47 +1,29 @@
 # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-taskdefinition.html
+# This is a custom template with some components derived from proper StackJack templates
```

Comment:
one note on verbiage: stackjack is a language that [gets plugged into templates and interpolated](https://github.com/WPMedia/stackjack/tree/main#why-do-we-need-stackjack) rather than a source of templates itself. the shared templates stackjack can plug into come from the [aws-templates](https://github.com/WPMedia/aws-templates/tree/master/templates) and [inf](https://github.com/WPMedia/aws-news-engineering-inf/tree/main/cfn/templates) repos. a rough analogy to a django project would be that stackjack is like jinja2 and aws-templates + inf are like [the base HTML templates](https://github.com/WPMedia/policeshootings/blob/develop/shootingsdb/templates/base.html).

Reply:
I don't think I follow. I thought whenever we referred to "stackjack templates" we were talking about the templates that stackjack ingests — kinda of how cloudformation templates are ingested by cloudformation???

Reply:
yeah on reflection this seems fine. to add some pedantic context, the way i think about it is that historically we've used "stackjack" as shorthand for the templating system that comprises stackjack + inf and/or aws-templates. the more precise way to think about it IMO is that the inf and aws-templates files ingest stackjack values, and cloudformation then ingests the files generated by interpolating the stackjack config values into the inf or aws-templates file.